demo/demo.jsx

```diff
@@ -71,7 +72,7 @@ export default class App extends React.Component {
           <VictoryAxis
             label="time axis"
             padding={{left: 10, right: 80}}
-            scale={"time"}
+            scale={d3Scale.time()}
```

Comment:
actually no, with the most recent refactor, `d3-scale` is not a dependency of `victory-axis`. It is now possible to pass a string scale, and have `victory-util` interpret that into a d3-scale. If a user would like to pass a `d3-scale` function anyhow, that will now be something they would include in their app.